app/javascript/courses/curricula/components/CoursesCurriculum__SubmissionsAndFeedback.re

```diff
@@ -128,11 +129,18 @@ let submissions =
            "Details about your submission on "
            ++ (submission |> Submission.createdAtPretty)
          }>
-         <div
-           className="text-xs font-semibold bg-gray-100 inline-block px-3 py-1 ml-2 rounded-t-lg border-t border-r border-l text-gray-800 leading-tight">
-           {"Submitted on "
-            ++ (submission |> Submission.createdAtPretty)
-            |> str}
+         <div className="flex justify-between items-end">
+           <h2 className="ml-2 mb-2 font-semibold text-sm lg:text-base leading-tight">
+             {"Submission #"
+              ++ (List.length(submissions) - index |> string_of_int)
+              |> str}
+           </h2>
+           <div
+             className="text-xs font-semibold bg-gray-100 inline-block px-3 py-1 mr-2 rounded-t-lg border-t border-r border-l text-gray-800 leading-tight">
+             {"Submitted on "
+              ++ (submission |> Submission.createdAtPretty)
+              |> str}
```

Comment:
This looks a bit tight on small screens. How about we hide the leading part of the date string?

```suggestion
             <span className="hidden md:inline">
               {str("Submitted on ")}
             </span>
             {Submission.createdAtPretty(submission) |> str}
```
![smaller_on_mobile](https://user-images.githubusercontent.com/98546/87028333-1ebf6300-c1fc-11ea-8ac1-d34758d2ae99.png)